providers/fab/src/airflow/providers/3rd-party-licenses/LICENSES-ui.txt

```diff
@@ -1,89 +0,0 @@
-Apache Airflow
```

Comment:
I think this one should not be removed ?

Reply:
Should this be at `providers/fab/src/airflow/providers/3rd-party-licenses`, or `providers/fab/3rd-party-licenses`? I checked and it's at the latter (I think it was moved at some point). But if it should be here instead will change.

Reply:
Re-placed